sw/ground_segment/cockpit/live.ml

```diff
@@ -1521,6 +1532,45 @@ let get_shapes = fun (geomap:G.widget)_sender vs ->
 let listen_shapes = fun (geomap:G.widget) ->
   safe_bind "SHAPE" (get_shapes geomap)
 
+let get_cam_status = fun (geomap:MapCanvas.widget) _sender vs ->
```

Comment:
The point the camera is pointing at (taking into account the bank angles and the ground alt) is already computed in the server program: https://github.com/paparazzi/paparazzi/blob/master/sw/ground_segment/tmtc/server.ml#L165
Your current position and this cam point should be enough to compute your polygon without extra information (maybe AGL).
Also be careful that at large bank angle, the projection of the footprint may send some corner very "far". I don't know how GTK canvas are dealing with this.